lib/lurker/form_builder.rb

```diff
@@ -11,55 +11,56 @@ def html
 
     private
 
-    def add_to_buffer(params, parent_labels = [])
-      params.each do |label, value|
-        if parent_labels.present?
-          label = "[#{label}]"
-        end
-
-        new_parent_labels = parent_labels.clone << label
+    def add_to_buffer(params, parent_accessors = [])
+      params.each do |name, value|
 
+        accessors = parent_accessors.clone << name
         if value.is_a?(Hash)
-          add_legend_to_buffer(parent_labels, label)
+          # add_legend_to_buffer(parent_labels, label)
```

Comment:
Then it's logical to remove `add_legend_to_buffer` method and it's template altogether. Is it ok?